app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -54,63 +58,73 @@ sealed trait Operation {
   val right: String
   val label: String
 
-  def eval(labels: Labels) : Labels
+  def eval(labels: Labels): Labels
 
-  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg){ref => labels.value(ref).getOrElse("")}
+  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg) { ref =>
+    labels.value(ref).getOrElse("")
+  }
 
   def toPlainString(value: BigDecimal): String = value.bigDecimal.toPlainString
```

Comment:
dont this this is required, doesnt add anything over the underlying

Reply:
Removed.